src/Microsoft.DocAsCode.Metadata.ManagedReference.Common/Parsers/TripleSlashCommentModel.cs

```diff
@@ -558,7 +558,25 @@ private void ResolveCrefLink(XNode node, string nodeSelector, Action<string, str
                             // When see and seealso are top level nodes in triple slash comments, do not convert it into xref node
                             if (item.Parent?.Parent != null)
                             {
-                                var replacement = XElement.Parse($"<xref href=\"{HttpUtility.UrlEncode(id)}\" data-throw-if-not-resolved=\"false\"></xref>");
+                                XElement replacement;
+                                if(string.IsNullOrEmpty(item.Value))
+                                {
+                                    replacement = XElement.Parse($"<xref href=\"{HttpUtility.UrlEncode(id)}\" data-throw-if-not-resolved=\"false\"></xref>");
+                                }
+                                else
+                                {
+                                    StringBuilder text = new StringBuilder();
+                                    if(item.HasElements)
+                                    {
+                                        foreach (var textNode in item.Nodes())
```

Comment:
What if the node is a tree, like `<cref> 0 <b> 1 <i> 2 </i> 3 </b> 4 </cref>` ?

Reply:
The test does test this and works, there are a lot of extra spaces added. Would love some direction on solving that a bit better

Reply:
A depth-first traversal of the tree to concat all leafs node text?

Reply:
I don’t just want their text, but any html in that text

Reply:
We'll likely escape `?text` properties in render, does `?text=<b>a</b>` render as `*b*` or `<b>a</b>` according to your test?

Reply:
`<b>` gets changed to `<strong>` and this gets encoded into the url. It would have made more sense if text goes inside the xref. ie `<xref ...>text is here</xref>`

Reply:
You're right it is going to escape the html :( Better to just go with Value then and let it strip the xml